core/src/main/scala/pureconfig/CollectionWriters.scala

```diff
@@ -41,7 +41,7 @@ trait CollectionWriters {
 
   implicit def mapWriter[T](implicit configConvert: Derivation[ConfigWriter[T]]) = new ConfigWriter[Map[String, T]] {
     override def to(keyVals: Map[String, T]): ConfigValue = {
-      ConfigValueFactory.fromMap(keyVals.mapValues(configConvert.value.to).asJava)
+      ConfigValueFactory.fromMap(keyVals.mapValues(configConvert.value.to).toMap.asJava)
```

Comment:
Just out curiosity, why is the `.toMap` helpful?

Reply:
In 2.13 I get this error:

```
[error] /Users/rui/Documents/Dev/own/pureconfig/core/src/main/scala/pureconfig/CollectionWriters.scala:44:76: type mismatch;
[error]  found   : Iterable[(String, com.typesafe.config.ConfigValue)]
[error]  required: java.util.Map[String, ?]
[error]  Note: implicit method mapWriter is not applicable here because it comes after the application point and it lacks an explicit result type
[error]       ConfigValueFactory.fromMap(keyVals.mapValues(configConvert.value.to).asJava)
```

I believe that `mapValues` now returns an `Iterable` instead of a `Map`.

Reply:
Oh, you're right: 2.13 [changes the return type of `mapValues`](https://www.scala-lang.org/api/2.13.0-RC1/scala/collection/Map.html#mapValues[W](f:V=%3EW):scala.collection.MapView[K,W]) to a `MapView`. Interesting. 

Sorry I didn't think to look before asking.